src/nxdoc/nuxeo-server/data-store/security/how-to-grant-the-edit-permission-without-the-remove-permission.md

```diff
@@ -88,19 +88,29 @@ As described on the page [Managing Permissions]({{page space='userdoc' page='per
 
 ## Procedure
 
-1.  [Add a new contribution]({{page page='how-to-contribute-to-an-extension'}}) to remove the `Remove` permission from `Write` permission.
+1.  [Add a new contribution]({{page page='how-to-contribute-to-an-extension'}}) to remove the `Remove` permission from `Write` permission. Check the [Nuxeo Explorer](http://explorer.nuxeo.com/nuxeo/site/distribution/Nuxeo%20Platform%20LTS%202019-10.10/viewExtensionPoint/org.nuxeo.ecm.core.security.SecurityService--permissions) page to update the suitable extension point.
 
     ```xml
       <extension target="org.nuxeo.ecm.core.security.SecurityService"
         point="permissions">
         <permission name="Write">
-          <remove>Remove</remove>
+          <include>AddChildren</include>
+          <include>WriteProperties</include>
+          <!-- The next line should be commented -->
+          <!--<include>Remove</include>-->
+          <include>ManageWorkflows</include>
+          <include>WriteLifeCycle</include>
+          <include>WriteVersion</include>
         </permission>
      </extension>
     ```
 
     This change will make the permission `ReadWrite`, displayed under the permission label "Edit" in the UI, act as wanted: it no longer includes the right to remove content.
 
+    {{#> callout type='warning' heading='Negative Permissions' }}
+    Negative permissions are disabled by default, due to Elasticsearch limitations (Check out the [Upgrade Notes to Nuxeo Platform 6.0]({{page space='nxdoc' page='upgrade-from-58-to-60'}}#migrating-negative-permissions))
```

Comment:
This isn't the same thing. The block you referenced on the upgrade page talks about negative acls which were deprecated due to ES not handling it. Here, the page is talking about creating/editing new permissions inside Nuxeo and removing a sub permission from one is still permitted and supported.

This should be reverted along with the contribution example. Otherwise, we may want to keep the addition below:
>Check the [Nuxeo Explorer](http://explorer.nuxeo.com/nuxeo/site/distribution/Nuxeo%20Platform%20LTS%202019-10.10/viewExtensionPoint/org.nuxeo.ecm.core.security.SecurityService--permissions) page to update the suitable extension point.

Reply:
OK! Thx!